Marlin/ultralcd.cpp

```diff
@@ -1110,13 +1110,24 @@ static void lcd_control_volumetric_menu() {
 #ifdef HAS_LCD_CONTRAST
   static void lcd_set_contrast() {
     if (encoderPosition != 0) {
-      lcd_contrast -= encoderPosition;
-      lcd_contrast &= 0x3F;
+      #ifdef U8GLIB_LM6059_AF
+        lcd_contrast += encoderPosition;
```

Comment:
@eboston Finally got those REVERSE options for the menu into the code. At the moment, trying to resolve an issue related to LCD contrast on graphical displays. I wonder if you wouldn't mind adding your 2¢ to the conversation, at #3912.